wds-required-plugins.php

```diff
@@ -112,11 +112,6 @@ public function required_text_markup() {
 	 * @return array
 	 */
 	public function filter_plugin_links( $actions = array(), $plugin ) {
-		// Remove edit link for all plugins
-		if ( array_key_exists( 'edit', $actions ) ) {
-			unset( $actions['edit'] );
-		}
-
```

Comment:
Question at that point is, do we want to apply it to the required plugins? While the editor is and always will be evil and bad, not sure this is the best place to stop editing of the plugins. Granted I originally wrote that piece that I just removed, but it was before it was a standalone plugin and was just part of wd_s.

In my perfect world I'd rather see it be standard process just to deactivate the editor as a whole across our sites.